glitch/docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '3'
+services:
+
+  #PHP Service
+  glitch:
+    image: fishnet/glitch
+    container_name: glitch-php
+    restart: unless-stopped
+    tty: true
+    working_dir: /var/www
+    env_file:
+      - ./.env
+    volumes:
+      - ./:/var/www
+      - node_modules:/var/www/node_modules
+      - ./config/php.ini:/usr/local/etc/php/conf.d/settings.ini
+    networks:
+      - glitch-network
+    depends_on:
+      - db
+
+  webserver:
+    image: nginx:alpine
+    container_name: glitch-nginx
+    restart: unless-stopped
+    ports: 
+      - "80:80"
+    volumes:
+      - ./:/var/www
+      - ./config/nginx.conf:/etc/nginx/conf.d/default.conf
+      - node_modules:/var/www/node_modules
+    networks:
+      - glitch-network
+    depends_on:
+      - db
+      - glitch 
+
+  # MySQL Service
+  db:
+    image: mariadb
+    container_name: glitch-mysql
+    restart: unless-stopped
+    command: --sql_mode=""
+    environment:
+      MYSQL_ROOT_PASSWORD: $DB_PASSWORD
+      MYSQL_DATABASE: $DB_DATABASE 
+      MYSQL_USER: $DB_USERNAME 
+      MYSQL_PASSWORD: $DB_PASSWORD
+    volumes:
+      - ./schema.sql:/docker-entrypoint-initdb.d/setup.sql
+      - dbdata:/var/lib/mysql
+    networks:
+      - glitch-network
```

Comment:
Glitch doesn't use a database, it's purely PHP + HTML. No point in running a database for this project